front/pages/api/w/[wId]/data_sources/[name]/managed/permissions/index.ts

```diff
@@ -82,7 +82,7 @@ async function handler(
     });
   }
 
```

Comment:
Actually we want to move this in GET and have builder in POST

I thiiibk we actually want Admin in POST